internal/tarjan/tarjan.go

```diff
@@ -24,13 +24,13 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-package plan
+package tarjan
```

Comment:
I suggest we call this package `sortutil` in case we add more sorting utilities here (and it's similar naming to the existing `strutil` and the like).